env.go

```diff
@@ -83,10 +84,15 @@ func (e *EnvironmentLoader) PrintEnvs(s interface{}) {
 	strctMap := strct.Map()
 	prefix := e.getPrefix(strct)
 
-	for key, val := range strctMap {
-		field := strct.Field(key)
+	keys := make([]string, 0, len(strctMap))
```

Comment:
just fyi. Structs package has `.Names()` function where it returns the field names.